vite.md

```diff
@@ -410,6 +411,25 @@ https://cdn.example.com/build/assets/app.9dce8d17.js
 
 Remember that [absolute URLs are not re-written by Vite](#url-processing), so they will not be prefixed.
 
+<a name="override-build-path"></a>
+## Override Build Path
+
+You can customise the folder name which vite uses when compiling. The example will change the folder name from 'build' to just 'folderName'
+
+```js
+import { defineConfig } from 'vite';
+import laravel from 'laravel-vite-plugin';
+
+export default defineConfig({
+    plugins: [
+        laravel({
+            // Default is 'build' 
+            buildPath: 'folderName'
```

Comment:
This should be `buildDirectory`

Reply:
Yup you are right.. no idea where/why I did build path .